testbed/tests/testdata/agent-config.yaml

```diff
@@ -10,6 +10,8 @@ exporters:
     endpoint: "127.0.0.1:56565"
 
 processors:
+  batch:
```

Comment:
This does not actually enable the processors. You need to also list them in the `processors` key of the pipeline.

Reply:
Thanks!

Reply:
I should have noticed the perf numbers didn't change 😮

Reply:
The use of the batcher needs a more careful change since it introduces a delay on the arrival of spans that breaks the basic tests. Edited the title and description to reflect that.